code/game/objects/items/devices/scanners/gas_analyzer.dm

```diff
@@ -165,9 +167,12 @@
 			message += span_notice("Temperature: [round(temperature - T0C,0.01)] &deg;C ([round(temperature, 0.01)] K)")
 			message += span_notice("Volume: [volume] L")
 			message += span_notice("Pressure: [round(pressure, 0.01)] kPa")
+			message += span_notice("Heat Capacity: [display_joules(heat_capacity)] / K")
+			message += span_notice("Thermal Energy: [display_joules(thermal_energy)]")
```

Comment:
Do we track this as joules in a player facing capacity, and if so should this list it's unit?

Reply:
display joules has its own logic, dont worry about it